### ISSUE TITLE
Can't get the sklearn-iris KfServing sample to work - InferenceService has no status

### ISSUE THREAD

**Issue by ofir-bt** [kubeflow/kfserving#1016]
/kind bug

I'm trying to run the sklearn-iris sample on my local Mac, using Minikube, with no success.

I've installed KubeFlow on my mac using:
`
kfctl apply -V -f kfctl_k8s_istio.v1.0.2.yaml
`

And then created a test namespace and deployed the sklearn-iris sample to it like that:
`kubectl create namespace test1`
`kubectl label namespace test1 serving.kubeflow.org/inferenceservice=enabled`
`kubectl -n test1 apply -f src/kfserving/docs/samples/sklearn/sklearn.yaml`

**The problem:**

The InferenceService doesn't get deployed:
`kubectl -n test1 get inferenceservice/sklearn-iris`
Gives me the output:

> NAME           URL   READY   DEFAULT TRAFFIC   CANARY TRAFFIC   AGE
> sklearn-iris                                                                                                  10m

Note that there's no values for "url", "Ready", etc...
The only values are for "Name" and "Age".

And:
Running:
`kubectl -n test1 describe inferenceservice/sklearn-iris`
Returns:

> Name:         sklearn-iris
> Namespace:    test1
> Labels:       <none>
> Annotations:  API Version:  serving.kubeflow.org/v1alpha2
> Kind:         InferenceService
> Metadata:
>   Creation Timestamp:  2020-08-11T14:45:57Z
>   Generation:          1
>   Resource Version:    16022
>   Self Link:           /apis/serving.kubeflow.org/v1alpha2/namespaces/test1/inferenceservices/sklearn-iris
>   UID:                 21d17258-f995-4d2b-a0db-96f3c70f0341
> Spec:
>   Default:
>     Predictor:
>       Min Replicas:  1
>       Sklearn:
>         Resources:
>           Limits:
>             Cpu:     1
>             Memory:  2Gi
>           Requests:
>             Cpu:          1
>             Memory:       2Gi
>         Runtime Version:  0.2.2
>         Storage Uri:      gs://kfserving-samples/models/sklearn/iris
> Status:
> Events:  <none>

I used the kfctl_k8s_istio.v1.0.2.yaml files from this repo: "https://github.com/kubeflow/manifests.git", from the "master" branch. Using this file from other branches gave me the same results.

**Environment:**

I'm not sure what are my Istio, Knative, KfServing versions, I guess they are listed in the kfctl_k8s_istio.v1.0.2.yaml file.
If anyone needs these versions, please comment and I'll Google for how to get these versions, or tell me how to get them if you know that.

- Istio Version:
- Knative Version:
- KFServing Version:
- Kubeflow version:
- Kfdef:kfctl_k8s_istio.v1.0.2.yaml
- Minikube version: v1.12.1
- Kubernetes version: (use `kubectl version`):
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.6"
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.3"
- OS (e.g. from `/etc/os-release`): Mac 10.13.6

**Comment by yuzisun** [comment]
@ofir-bt Looks it is the same issue as https://github.com/kubeflow/kfserving/issues/951, if you can downgrade to 1.17 that should work

**Comment by ofir-bt** [comment]
Thanks @yuzisun, after downgrading K8S to 1.17, the error is resolved.
